services/gitea/gitea-release.service.js

```diff
@@ -0,0 +1,147 @@
+import Joi from 'joi'
+import { optionalUrl } from '../validators.js'
+import { latest, renderVersionBadge } from '../version.js'
+import { NotFound, pathParam, queryParam } from '../index.js'
+import { documentation, httpErrorsFor } from './gitea-helper.js'
+import GiteaBase from './gitea-base.js'
+
+const schema = Joi.array().items(
+  Joi.object({
+    name: Joi.string().required(),
+    tag_name: Joi.string().required(),
+    prerelease: Joi.boolean().required(),
+  }),
+)
+
+const sortEnum = ['date', 'semver']
+const displayNameEnum = ['tag', 'release']
+const dateOrderByEnum = ['created_at', 'published_at']
+
+const queryParamSchema = Joi.object({
+  gitea_url: optionalUrl.required(),
+  include_prereleases: Joi.equal(''),
+  sort: Joi.string()
+    .valid(...sortEnum)
+    .default('date'),
+  display_name: Joi.string()
+    .valid(...displayNameEnum)
+    .default('tag'),
+  date_order_by: Joi.string()
+    .valid(...dateOrderByEnum)
+    .default('created_at'),
+}).required()
+
+export default class GiteaRelease extends GiteaBase {
+  static category = 'version'
+
+  static route = {
+    base: 'gitea/v/release',
+    pattern: ':user/:repo',
+    queryParamSchema,
+  }
+
+  static openApi = {
+    '/gitea/v/release/{user}/{repo}': {
+      get: {
+        summary: 'Gitea Release',
+        description: documentation,
+        parameters: [
+          pathParam({
+            name: 'user',
+            example: 'forgejo',
+          }),
+          pathParam({
+            name: 'repo',
+            example: 'forgejo',
+          }),
+          queryParam({
+            name: 'gitea_url',
+            example: 'https://codeberg.org',
+            required: true,
+          }),
+          queryParam({
+            name: 'include_prereleases',
+            schema: { type: 'boolean' },
+            example: null,
+          }),
+          queryParam({
+            name: 'sort',
+            schema: { type: 'string', enum: sortEnum },
+            example: 'semver',
+          }),
+          queryParam({
+            name: 'display_name',
+            schema: { type: 'string', enum: displayNameEnum },
+            example: 'release',
+          }),
+          queryParam({
+            name: 'date_order_by',
```

Comment:
@CanisHelix late to the game on this one, but I happened to stumble upon this file. It doesn't look this query parameter or the corresponding `orderBy` parameter lower down is used, am I missing something?